docs/src/components/storytelling/cli-video.tsx

```diff
@@ -1,18 +1,17 @@
 export default function CliVideo() {
   return (
-    <section className="relative bg-slate-3">
-      <div className="mx-auto max-w-4xl px-6 py-6 sm:pb-12 lg:pb-16">
-        <div className="text-center">
-          <h2 className="text-3xl font-bold sm:text-4xl">
-            Two-way editing, effortlessly
+    <section className="relative bg-white">
+      <div className="mx-auto grid max-w-5xl grid-cols-8 gap-6 gap-y-10 px-6 py-12 md:grid-cols-12">
+        <div className="col-span-8 text-center sm:col-span-6 sm:col-start-2 md:col-start-1 md:col-end-6 md:mt-8 md:pr-6 md:text-left">
+          <h2 className="mb-4 text-2xl font-medium">
+            Two-way editing, effortlessly with the Keystatic CLI
```

Comment:
this seems to be repeating copy below: Create a new Next.js or Astro project in seconds with **the Keystatic CLI**

is that ok?